src/commands/fortune.js

```diff
@@ -0,0 +1,65 @@
+import { exampleCommand, beep, withStar } from '../utilities/string'
+import { sendError } from '../utilities/message'
+
+import { getOneCardData, getMajorCards } from '../services/tarot'
+import fortuneInfo from '../templates/fortuneInfo'
+
+const error = [
+  beep('You may give bubeep a wrong type.'),
+  exampleCommand('"tarot normal" or "tarot single"'),
+]
+
+const drawCard = async () => {
+  const cardNumber = Math.floor(Math.random() * 22)
+  const isReversed = Math.floor(Math.random() >= 0.5)
+  const cardData = await getOneCardData(cardNumber, isReversed)
+  return { ...cardData }
+}
+
+const drawManyCard = async amount => {
+  const cardDataDict = {}
```

Comment:
I would prefer another solution a 555555

any of these choices will be so เท่ na
- write a function that returns an array of random int (no repeat) first, then map the array with card data later
- write a function that [shuffle](https://javascript.info/task/shuffle) an entire array, then shuffle all card, and then get the first 3 (or any number) cards with `slice`